src/schema/v2/sale/index.ts

```diff
@@ -231,6 +231,11 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ buyers_premium }) => !!buyers_premium,
       },
+      isLotConditionsReportEnabled: {
```

Comment:
thoughts on testing this field more directly in the sale test? It's inadvertently tested via default values, but there's seems to be a precedent for even testing [trivial fields, like `timeZone`, directly](https://github.com/artsy/metaphysics/blob/73d40d98f13dab5b2fe9f0f2d6742fe081d351b4/src/schema/v2/sale/__tests__/index.test.js#L26-L42).

Reply:
Having a single `describe` for each field would lead a very verbose test file. I don't think we should deliberately extract it out.